docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
 Registers tool with the given group and start and stop tool trigger text
 
 **OPTIONS**  
- -C, --config PATH  
-Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+ `-C`, `--config` PATH  
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+Path to the pbench-agent configuration file.
 
--g, --group,--groups STR  
+`-g`, `--group`, `--groups` STR  
 Registers the trigger in the <group-name> group. If no group is specified, the 'default'
 
---start-trigger STR  
+`--start-trigger` STR  
 [required]
 
---stop-trigger STR  
+`--stop-trigger` STR  
 [required]
 
---help  
+`--help`  
 Show this message and exit.
 
+
 ---
 
-### pbench-results-move
+##### pbench-user-benchmark
 
 ---
 
 **NAME**  
-pbench-results-move, Move results tarball to the server
+`pbench-user-benchmark`, Control start/stop/post-process-tools.
 
 **SYNOPSIS**  
-`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]
-`
+`pbench-user-benchmark[-C][--tool-group][--iteration-list][--sysinfo][--pbench-pre][--pbench-post][--use-tool-triggers][--no-stderr-capture] --<command to run>`
```

Comment:
I'm pretty sure that there should be a blank space between the `--` and `<command-to-run>`.  (And, it's best not to have any blank spaces between the `<` and the `>` in placeholder tokens -- use hyphens or underscores (but pick one and be consistent throughout the documentation)).